packages/react-super-components/src/components/msc-tile/msc-tile.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import i18next from 'i18next';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { OsdsText } from '@ovhcloud/ods-components/text/react';
+import { OsdsChip } from '@ovhcloud/ods-components/chip/react';
+import { OsdsTile } from '@ovhcloud/ods-components/tile/react';
+import { OsdsLink } from '@ovhcloud/ods-components/link/react';
+import { OsdsIcon } from '@ovhcloud/ods-components/icon/react';
+
+import { ODS_ICON_NAME, ODS_ICON_SIZE } from '@ovhcloud/ods-components/icon';
+import { ODS_TEXT_LEVEL, ODS_TEXT_SIZE } from '@ovhcloud/ods-components/text';
+import { useDynamicTranslation } from '../../translation/useDynamicTranslation';
+
+interface Badge {
+  text: string;
+  color: ODS_THEME_COLOR_INTENT;
+}
+
+export interface ScTileProps {
+  href?: string;
+  isExternalHref?: boolean;
+  imgSrc?: string;
+  imgAlt?: string;
+  texts: {
+    title: string;
+    description: string;
+    category: string;
+  };
```

Comment:
Difficult to understand what is the proper way to structure props: 

It could be `imgSrc` & `imgAlt`, why not `img: { src, alt }` ? 
it could be `texts: { title, description, category }` but why not `title`, `description` & `category` ?

Reply:
Yes good catch i decide to split also `img: { src, alt }` for me it's better to group props by logic to avoid a component with 30 props, but i am open to discussion if you prefer another way